### PR TITLE
Backport https://github.com/decidim/decidim/pull/7028

### PR DESCRIPTION
**Pull request by armandfardeau**

### CODE REVIEW

**Comment by armandfardeau** [comment]
Add FR translation

File: decidim-initiatives/config/locales/fr.yml [hunk 1]
```diff
@@ -96,10 +96,14 @@ fr:
         configure: Configure
         confirm_destroy: Confirm destroy
         destroy: Destroy
+        export: Export
```
Comment (armandfardeau) [inline decidim-initiatives/config/locales/fr.yml:99]:
```suggestion
        export: Exporter
```

File: decidim-initiatives/config/locales/fr.yml [hunk 2]
```diff
@@ -96,10 +96,14 @@ fr:
...
         new: New
         new_initiative_type: Nouveau type d'initiative
         publish: Publish
         unpublish: Unpublish
+      exports:
+        export_as: Export as
```
Comment (armandfardeau) [inline decidim-initiatives/config/locales/fr.yml:105]:
```suggestion
        export_as: Export comme
```

File: decidim-initiatives/config/locales/fr.yml [hunk 3]
```diff
@@ -165,6 +169,17 @@ fr:
         initiatives_types: Types d'initiative
     events:
       initiatives:
+        admin:
+          initiative_sent_to_technical_validation:
+            email_intro: The initiative "%{resource_title}" has been sent to technical
+              validation. Check it out at <a href="%{admin_initiative_url}">the admin
+              panel</a>
```
Comment (armandfardeau) [inline decidim-initiatives/config/locales/fr.yml:176]:
```suggestion
            email_intro: L'initiative "%{resource_title}" a été envoté à la validation technique. Consultez l'esapce <a href="%{admin_initiative_url}">d'administration</a>
```

File: decidim-initiatives/config/locales/fr.yml [hunk 4]
```diff
@@ -282,6 +311,7 @@ fr:
           edit:
             accept: Accepter l'initiative
             confirm: Êtes-vous sûr(e) ?
+            confirm_send_to_technical_validation: Are you sure?
```
Comment (armandfardeau) [inline decidim-initiatives/config/locales/fr.yml:314]:
```suggestion
            confirm_send_to_technical_validation: Êtes-vous sûr(e) ?
```

File: decidim-initiatives/config/locales/fr.yml [hunk 5]
```diff
@@ -271,6 +298,8 @@ fr:
             no_members_yet: Il n'y a pas encore de membres dans le Comité de promotion
             revoke: Révoquer
             title: Membres du comité de promotion
+          revoke:
+            success: Request has been revoked.
```
Comment (armandfardeau) [inline decidim-initiatives/config/locales/fr.yml:302]:
```suggestion
            success: La requête a été révoquée
```

File: decidim-initiatives/config/locales/fr.yml [hunk 6]
```diff
@@ -378,10 +408,14 @@ fr:
         filter_type_values:
           all: Tous
       committee_requests:
+        approve:
+          success: Success
```
Comment (armandfardeau) [inline decidim-initiatives/config/locales/fr.yml:412]:
```suggestion
          success: Succès
```

File: decidim-initiatives/config/locales/fr.yml [hunk 7]
```diff
@@ -378,10 +408,14 @@ fr:
...
         new:
           continue: Continuer
           help_text: Vous êtes sur le point de demander à devenir membre du Comité
             de promotion de cette initiative
+        revoke:
+          success: Success
```
Comment (armandfardeau) [inline decidim-initiatives/config/locales/fr.yml:418]:
```suggestion
          success: Succès
```

File: decidim-initiatives/config/locales/fr.yml [hunk 8]
```diff
@@ -523,6 +588,14 @@ fr:
           hidden_authors_count:
             one: et 1 autre personne
             other: et %{count} autres personnes
+        committee_members:
+          approve: Approve
```
Comment (armandfardeau) [inline decidim-initiatives/config/locales/fr.yml:592]:
```suggestion
          approve: Approuver
```

File: decidim-initiatives/config/locales/fr.yml [hunk 9]
```diff
@@ -523,6 +588,14 @@ fr:
...
+          confirm_revoke: Are you sure?
```
Comment (armandfardeau) [inline decidim-initiatives/config/locales/fr.yml:593]:
```suggestion
          confirm_revoke: Êtes-vous sûr(e) ?
```

File: decidim-initiatives/config/locales/fr.yml [hunk 10]
```diff
@@ -523,6 +588,14 @@ fr:
...
+          invite_to_committee_help: Share this link to invite other participants to
+            the promoter committee
+          no_members_yet: There are no members in the promoter committee
+          revoke: Revoke
```
Comment (armandfardeau) [inline decidim-initiatives/config/locales/fr.yml:597]:
```suggestion
          revoke: Revoquer
```

File: decidim-initiatives/config/locales/fr.yml [hunk 11]
```diff
@@ -523,6 +588,14 @@ fr:
           hidden_authors_count:
             one: et 1 autre personne
             other: et %{count} autres personnes
+        committee_members:
+          approve: Approve
+          confirm_revoke: Are you sure?
+          invite_to_committee_help: Share this link to invite other participants to
+            the promoter committee
+          no_members_yet: There are no members in the promoter committee
```
Comment (armandfardeau) [inline decidim-initiatives/config/locales/fr.yml:596]:
```suggestion
          no_members_yet: Il n'y a pas de membre dans le comité de promotion
```

File: decidim-initiatives/config/locales/fr.yml [hunk 12]
```diff
@@ -523,6 +588,14 @@ fr:
...
+          revoke: Revoke
+          title: Committee members
```
Comment (armandfardeau) [inline decidim-initiatives/config/locales/fr.yml:598]:
```suggestion
          title: Membre du comité de promotion
```

File: decidim-initiatives/config/locales/fr.yml [hunk 13]
```diff
@@ -523,6 +588,14 @@ fr:
           hidden_authors_count:
             one: et 1 autre personne
             other: et %{count} autres personnes
+        committee_members:
+          approve: Approve
+          confirm_revoke: Are you sure?
+          invite_to_committee_help: Share this link to invite other participants to
+            the promoter committee
```
Comment (armandfardeau) [inline decidim-initiatives/config/locales/fr.yml:595]:
```suggestion
          invite_to_committee_help: Partagez ce lien pour inviter d'autres participants au comité de promotion
```

File: decidim-initiatives/config/locales/fr.yml [hunk 14]
```diff
@@ -165,6 +169,17 @@ fr:
         initiatives_types: Types d'initiative
     events:
       initiatives:
+        admin:
+          initiative_sent_to_technical_validation:
+            email_intro: The initiative "%{resource_title}" has been sent to technical
+              validation. Check it out at <a href="%{admin_initiative_url}">the admin
+              panel</a>
+            email_outro: You have received this notification because you are an admin
```
Comment (paulinebessoles) [inline decidim-initiatives/config/locales/fr.yml:177]:
```suggestion
            email_outro: Vous avez reçu cette notification car vous êtes administrateur de la plateforme
```

File: decidim-initiatives/config/locales/fr.yml [hunk 15]
```diff
@@ -165,6 +169,17 @@ fr:
...
+              of the platform.
```
Comment (paulinebessoles) [inline decidim-initiatives/config/locales/fr.yml:178]:
```suggestion
```

File: decidim-initiatives/config/locales/fr.yml [hunk 16]
```diff
@@ -165,6 +169,17 @@ fr:
...
+            email_subject: Initiative "%{resource_title}" was sent to technical validation.
```
Comment (paulinebessoles) [inline decidim-initiatives/config/locales/fr.yml:179]:
```suggestion
            email_subject: La pétition "%{resource_title}" a été envoyée à la validation technique
```

File: decidim-initiatives/config/locales/fr.yml [hunk 17]
```diff
@@ -165,6 +169,17 @@ fr:
...
+            notification_title: The initiative "%{resource_title}" has been sent to
```
Comment (paulinebessoles) [inline decidim-initiatives/config/locales/fr.yml:180]:
```suggestion
            notification_title: La pétition "%{resource_title}" a été envoyée à la validation technique. Consultez l'espace <a href="%{admin_initiative_url}">d'administration</a>
```

File: decidim-initiatives/config/locales/fr.yml [hunk 18]
```diff
@@ -165,6 +169,17 @@ fr:
...
+              technical validation. Check it out at <a href="%{admin_initiative_path}">the
```
Comment (paulinebessoles) [inline decidim-initiatives/config/locales/fr.yml:181]:
```suggestion

```

File: decidim-initiatives/config/locales/fr.yml [hunk 19]
```diff
@@ -165,6 +169,17 @@ fr:
...
+              admin panel</a>
```
Comment (paulinebessoles) [inline decidim-initiatives/config/locales/fr.yml:182]:
```suggestion

```

File: decidim-initiatives/config/locales/fr.yml [hunk 20]
```diff
@@ -263,6 +288,8 @@ fr:
             initiatives: Initiatives
             state: Etat
         committee_requests:
+          approve:
+            success: Request has been approved.
```
Comment (paulinebessoles) [inline decidim-initiatives/config/locales/fr.yml:292]:
```suggestion
            success: La demande a été approuvée avec succès.
```

File: decidim-initiatives/config/locales/fr.yml [hunk 21]
```diff
@@ -463,6 +497,16 @@ fr:
...
+        approve_membership_request:
+          email_intro: "%{author_nickname} accepted your application to be part of
+            the promoter committee for the initiative %{resource_title}."
+          email_outro: 'You received this notification because you applied to this
+            initiative: %{resource_title}'
+          email_subject: "%{author_nickname} accepted your application to the promoter
+            committee"
+          notification_title: <a href="%{author_profile_url}">%{author_nickname}</a>
+            accepted your application to be part of the promoter committee for the
+            following initiative <a href="%{resource_url}">%{resource_title}</a>.
```
Comment (paulinebessoles) [inline decidim-initiatives/config/locales/fr.yml:509]:
```suggestion
        approve_membership_request:
          email_intro: "%{author_nickname} a accepté votre demande pour rejoindre le comité de promotion de la pétition %{resource_title}."
          email_outro: 'Vous avez reçu cette notification car vous avez demandé à rejoindre le comité de promotion de cette pétition : %{resource_title}'
          email_subject: "%{author_nickname} a accepté votre demande de rejoindre le comité de promotion"
          notification_title: <a href="%{author_profile_url}">%{author_nickname}</a> a accepté votre demande de rejoindre le comité de promotion de la pétition <a href="%{resource_url}">%{resource_title}</a>.
```

File: decidim-initiatives/config/locales/fr.yml [hunk 22]
```diff
@@ -479,6 +523,26 @@ fr:
...
+        revoke_membership_request:
+          email_intro: "%{author_nickname} rejected your application to be part of
+            the promoter committee for the following initiative %{resource_title}."
+          email_outro: 'You received this notification because you applied to this
+            initiative : %{resource_title}.'
+          email_subject: "%{author_nickname} rejected your application to the promoter
+            committee"
+          notification_title: <a href="%{author_profile_url}">%{author_nickname}</a>
+            rejected you application to be part of the promoter committee for the
+            following initiative <a href="%{resource_url}">%{resource_title}</a>.
```
Comment (paulinebessoles) [inline decidim-initiatives/config/locales/fr.yml:535]:
```suggestion
        revoke_membership_request:
          email_intro: "%{author_nickname} a rejeté votre demande pour rejoindre le comité de promotion de la pétition %{resource_title}."
          email_outro: 'Vous avez reçu cette notification car vous avez demandé à rejoindre le comité de promotion de cette pétition : %{resource_title}'
          email_subject: "%{author_nickname} a rejeté votre demande de rejoindre le comité de promotion"
          notification_title: <a href="%{author_profile_url}">%{author_nickname}</a> a rejeté votre demande de rejoindre le comité de promotion de la pétition <a href="%{resource_url}">%{resource_title}</a>.
```

File: decidim-initiatives/config/locales/fr.yml [hunk 23]
```diff
@@ -479,6 +523,26 @@ fr:
...
+        spawn_committee_request_event:
+          email_intro: "%{applicant_nickname} applied for the promoter committee of
+            your initiative %{resource_title}. To accept or reject the application,
+            go to the edit form of your initiative."
+          email_outro: 'You received this notification because you are the author
+            of this initiative : %{resource_title}'
+          email_subject: "%{applicant_nickname} wants to join your initiative"
+          notification_title: <a href="%{applicant_profile_url}">%{applicant_nickname}</a>
+            applied for the promoter committee of your initiative <a href="%{resource_url}">%{resource_title}</a>.
+            To accept or reject click <a href="%{resource_url}/edit">here</a>.
```
Comment (paulinebessoles) [inline decidim-initiatives/config/locales/fr.yml:545]:
```suggestion
        spawn_committee_request_event:
          email_intro: "%{applicant_nickname} a demandé à rejoindre le comité de promotion de la pétition %{resource_title}. Pour accepter ou rejeter cette demande, rendez-vous dans le formulaire d'édition de cette pétition."
          email_outro: 'Vous avez reçu cette notification car vous êtes l'auteur de cette pétition : %{resource_title}'
          email_subject: "%{applicant_nickname} a demandé à rejoindre le comité de promotion de votre pétition"
          notification_title: <a href="%{applicant_profile_url}">%{applicant_nickname}</a> a demandé à rejoindre le comité de promotion de votre pétition <a href="%{resource_url}">%{resource_title}</a>. Pour accepter ou rejeter, rendez-vous <a href="%{resource_url}/edit">sur le formulaire d'édition</a>.
```

File: decidim-initiatives/config/locales/fr.yml [hunk 24]
```diff
@@ -491,6 +555,7 @@ fr:
           help: S'il vous plaît, remplissez les champs suivants avec vos données personnelles
             pour signer l'initiative
           select_scope: Sélectionner un périmètre d'application
+          select_scope_label: Scope
```
Comment (paulinebessoles) [inline decidim-initiatives/config/locales/fr.yml:558]:
```suggestion
          select_scope_label: Secteur
```

**Review by Quentinchampenois — CHANGES_REQUESTED** [review]
Hi @armandfardeau thank you for your work, I agree with the @paulinebessoles's [suggestion](https://github.com/OpenSourcePolitics/decidim/pull/1095/files#r571997165).

Otherwise the backport seems valid to me!

**Review by moustachu — APPROVED** [review]

**Pull request merged**